docs/source/how_to/use_oauth2.rst

```diff
@@ -147,9 +147,59 @@ However, if the response is not successful, you’ll receive an error response.
 Using the access token
 ----------------------
 
-.. note::
+Now that you have the access token, you can get the user data by sending a
+``GET`` request to ``/open_auth/get_user_data/`` with the access token in the
+header as follows:
+
+.. code-block:: json
+
+  {
+    "Authorization": "Bearer vqygrcouTuyAYHZLz3rGcZf5FpPd3K"
+  }
+
+where ``vqygrcouTuyAYHZLz3rGcZf5FpPd3K`` is the access token.
+
+If the access token is valid, then you will receive a 200 response
+with a dictionary containing the user's information based on the scopes
+of the client app, similar to this:
+
+.. code-block:: json
+
+    {
+       "userId": 1234,
+       "username": "19114099",
+       "person":{
+          "shortName":"dhruvkb",
+          "fullName":"Dhruv Kanti Bhanushali",
+          "roles":[
+             {
+                "role":"Student",
+                "activeStatus":"ActiveStatus.IS_ACTIVE"
+             },
+             {
+                "role":"Maintainer",
+                "activeStatus":"ActiveStatus.IS_ACTIVE"
+             }
+          ],
+          ...
+       },
+       "student":{
+            "enrolmentNumber": "19114099",
+             ...
+       },
+       "contactInformation":{
+            "instituteWebmailAddress": "xyz@iitr.ac.in"
+             ...
+       }
+    }
+
+However, if the access token is invalid, you will receive the following error response.
+
+.. code-block::
+
+      "Associated application not found"
+
 
-  This part of the process is still under development. So check back later.
 
 Future plans
```

Comment:
Make sure to remove or edit this section also.
We may consider using a real apps' scope-based OAuth so the developer can interact with the Omniport apps or services APIs.

Reply:
I think that should be in a separate PR